### PR TITLE
chore: update readme with cli changes, add additional instructions

### PR DESCRIPTION
**Pull request by c3ppc3pp**
blocked by #248

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,25 +76,47 @@ Seamless testing whenever your team wants to test the application, without waiti
...
-dyrector.io's command-line interface (CLI) lets you run a complete dyrector.io development environment locally with the following services: UI Service (crux-ui), Backend Service (crux), PostgreSQL database, Authentication, Migrations, and SMTP mail server.
+2. Execute `go run ./cli up` in the project root
+3. Open the `localhost:8000` and use the platform
+
+dyrector.io's command-line interface (CLI) lets you run a complete dyrector.io development environment locally with the following services: UI Service (crux-ui), Backend Service (crux), PostgreSQL databases, Authentication, Migrations, and SMTP mail server.
+
+#### Using the platform with CLI for demonstration or testing
+
+1. Execute `go run ./cli up` in the project root
+2. After you navigated to `localhost:8000` (this is the default traefik port) you will see a loginscreen
```
Comment (robot9706) [inline README.md:102]: `Login screen` I think 🤔

File: README.md [hunk 2]
```diff
@@ -76,25 +76,47 @@ Seamless testing whenever your team wants to test the application, without waiti
...
+3. Register an account with whatever e-mail address you see fit (doesn't have to be valid one)
+4. Navigate to `localhost:4436` where you will find your mail as all outgoing e-mails will land here
+5. Open your e-mail message and use the link inside to activate your account
+6. Enjoy!
+
+#### Usint the platform with CLI for development
```
Comment (robot9706) [inline README.md:108]: `Using` :D

File: README.md [hunk 3]
```diff
@@ -76,25 +76,47 @@ Seamless testing whenever your team wants to test the application, without waiti
...
+
+1. Read the CLI documentation first(see the end of this section)
+2. Decide which part of the project you want to work on, in this case its crux, crux-ui or both
```
Comment (robot9706) [inline README.md:111]: `In this case it's`

File: README.md [hunk 4]
```diff
@@ -76,25 +76,47 @@ Seamless testing whenever your team wants to test the application, without waiti
...
+3. Modify the CLI's settings file if you need it.
+4. Execute the correct CLI command using the appropriate flags to turn off crux or crux-ui services
+5. Start crux or crux-ui with the appropriate `npm` command, ususally `npm run start`
```
Comment (robot9706) [inline README.md:114]: `usually` :D

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -76,25 +76,47 @@ Seamless testing whenever your team wants to test the application, without waiti
...
+6. After you navigated to `localhost:8000` (this is the default traefik port) you will see a loginscreen
```
Comment (bvektor) [inline README.md:115]: `login screen` Same as line 102

Comment (c3ppc3pp) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (c3ppc3pp) [reply README.md]: good catch, it was sus :D

File: README.md (hunk 3 above)
Comment (c3ppc3pp) [reply README.md]: :facepalm: epic fail, total embarrassment :(

fixed :)

File: README.md (hunk 1 above)
Comment (c3ppc3pp) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (c3ppc3pp) [reply README.md:108]: rephrased that part, thanks

**Review by robot9706 — APPROVED** [review]
LGTM 😎

**Comment by polaroi8d** [comment]
The proper type of this PR is `chore`, please use this.

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**